src/extension.ts

```diff
@@ -239,20 +239,37 @@ export function activate(context: vscode.ExtensionContext): void {
         void setCSSMirrorContentEnabled(context, !cssMirrorContent);
     }));
 
-    context.subscriptions.push(vscode.commands.registerCommand(`${SETTINGS_VIEW_NAME}.launchHtml`, (fileUri: vscode.Uri): void => {
+    context.subscriptions.push(vscode.commands.registerCommand(`${SETTINGS_VIEW_NAME}.launchHtml`, async (fileUri: vscode.Uri): Promise<void> => {
```

Comment:
could you also add a test for this scenario in:

https://github.com/microsoft/vscode-edge-devtools/blob/main/test/extension.test.ts

that way we will avoid future regressions.

Reply:
Done. Thanks!